precompiles/utils/src/precompile_set.rs

```diff
@@ -837,6 +838,68 @@ impl<A> IsActivePrecompile for RevertPrecompile<A> {
 	}
 }
 
+/// Precompiles that were removed from a precompile set.
+/// Still considered precompiles but are inactive and always revert.
+pub struct RemovedPrecompilesAt<A>(PhantomData<A>);
+impl<A> PrecompileSetFragment for RemovedPrecompilesAt<A>
+where
+	A: Get<Vec<H160>>,
```

Comment:
Possible optimisation: you can replace `Vec<H160>` by [H160; N] where N is a const generic

Reply:
Not sure if the worth it in terms of code complexity, will have to add the const N in many places. It is known that the Vector is slower in small list, but the different should not be noticeable